SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# How To Report a Vulnerability
+
+ If you think you have found a vulnerability in Jakarta EE Servlet you can report it using one of the following ways:
```

Comment:
As organization hosts many specification projects perhaps referencing Servlet only is not quite right.

Reply:
🤦 Good catch, thanks!